modules/bmo-performing-a-live-update-to-the-hostfirmwaresettings-resource.adoc

```diff
@@ -34,37 +35,38 @@ $ oc patch hostfirmwaresettings <hostname> --type merge -p \ <1>
 +
 [NOTE]
 ====
-Get the `FirmwareSchema` resource to determine which settings the hardware supports. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
+Get the `FirmwareSchema` resource to determine which settings the hardware supports and what settings and values you can update. You cannot update read-only values and you cannot update the `FirmwareSchema` resource. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the resource.
```

Comment:
Get the `FirmwareSchema` resource to determine which settings the hardware supports and what settings and values you can update. You cannot update read-only values and you cannot update the `FirmwareSchema` resource. You can also use `oc edit <hostname> hostfirmwaresettings -n openshift-machine-api` to update the `HostFirmwareSettings` resource. (added the resource name in the last part to clarify it's not FirmwareSchema we just said user cannot update)